IAMOTZ/ievents
# #152883888 Create the template for the  home page

#### What does this PR do?
Add the template for the application's home page.
#### Description of Task to be completed?
Create template home page(TEMPLATE)
#### How should this be manually tested?
- clone the repo
- cd into the template directory
- open the index.html with any browser of your choice
- the template should open in that browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152883888
#### Screenshots (if appropriate)
![just-now](https://user-images.githubusercontent.com/26048536/32842595-c4a48626-ca1d-11e7-97f1-e7e281ebfb5b.png)
#### Questions:
None
